README.md

```diff
@@ -1,16 +1,43 @@
-# mt32.net
+<img src="https://user-images.githubusercontent.com/39274150/111981613-30d1ff00-8b08-11eb-8767-b89f2e0970be.png"/>
+<h1>https://mt32.net</h1>
+<h2>The blog for all computer interested</h2>
 
+ Netlify  
+|----|
 [![Netlify Status](https://api.netlify.com/api/v1/badges/4e30a997-7aa8-4fa2-8856-93e9b9a5ee39/deploy-status)](https://app.netlify.com/sites/mtorials-test/deploys)
 
-A hugo website deployed with netlify!
+# About mt32.net
 
-## License
+`mt32.net` is running by @mtorials and @universum two hobby programmer. 
+Not only the owners share their knowledge on this blog. Everyone who has a github account can share their knowledge on this blog. 
+The blog focuses on technology, computers and programming. But everything the computer scientist heart desires is welcome!
+
+# Read Article 
+## Read interested Articel from the mt32.net community!
+![image](https://user-images.githubusercontent.com/39274150/111983170-1e58c500-8b0a-11eb-960a-54296d572d8e.png)
+
+# Write your own Article 
+## Write your own articles with markdown
+![image](https://user-images.githubusercontent.com/39274150/111983370-59f38f00-8b0a-11eb-9b6b-377c71e16d35.png)
+
+# How to write a article?
+
+Do you want make a article! Nice! Please read THIS file when you're new and do not know how.
+
+# Netifly
+
+mt32.net work with the deploy platform `Netifly`. Netifly can deploy and build web apps based on github workflows.

```

Comment:
Not true anymore. I will update the README myself accordingly in the next few days